app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt

```diff
@@ -0,0 +1,142 @@
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.scesi.cappuchino.R
+
+@Composable
+fun HomeScreen() {
+    Column(
+        modifier = Modifier
+//            .background(Color.Gray)
+            .fillMaxSize()
+            .padding(20.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Top
+
+    ) {
+        Text(
+            text = "Ponte comodo y arma tu horario con un cappuchino",
+            textAlign = TextAlign.Center,
+            fontSize = 15.sp,
+            modifier = Modifier
+                .wrapContentHeight()
+//                .background(Color.Cyan)
+                .padding(top = 20.dp)
+
+        )
+        Spacer(modifier = Modifier.height(16.dp))
+
+        Image(
+            painter = painterResource(id = R.drawable.cap),
+            contentDescription = "descrip",
+            modifier = Modifier
+                .size(40.dp)
+        )
+        SearchBar()
+        Spacer(modifier = Modifier.height(16.dp))
+        TextBoxCarreras(text= "Licenciatura en ing en sistemas", onClick={})
+        TextBoxCarreras(text= "Licenciatura en ing en Infirmatica", onClick={})
+        TextBoxCarreras(text= "Licenciatura en ing en indistrual", onClick={})
+        TextBoxCarreras(text= "Licenciatura en ing en electronica", onClick={})
+
+
+    }
+}
+
+@Composable
+fun TextBoxCarreras(
+    text: String,
+    onClick: () -> Unit)
+{
+
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .wrapContentHeight()
+            .padding(5.dp)
+            .background(
+                color = Color(0xFFD1D8FF),
+                shape = RoundedCornerShape(8.dp)
+            )
+            .clickable { onClick() }
+            .padding(horizontal = 6.dp, vertical = 12.dp),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween
+    ){
+        Text(
+            text = text,
+            fontSize = 15.sp,
+            color = Color.Black
+        )
+        Icon(
+            painter = painterResource(id = R.drawable.fecha),
+            contentDescription = "Arrow Icon",
+            tint = Color.Black,
+            modifier = Modifier.size(24.dp)
+        )
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SearchBar(){
+    var searchText by remember { mutableStateOf("") }
+
+    OutlinedTextField(
+        value = searchText,
+        onValueChange = { searchText = it },
+        placeholder = { Text("Busca tu carrera", fontSize = 10.sp) },
```

Comment:
Dont use raw strings, instead use string resources , extract it (option + enter)